src/main/java/uk/gov/hmcts/reform/prl/models/dto/citizen/CitizenDocuments.java

```diff
@@ -30,14 +30,16 @@ public class CitizenDocuments {
 
     // Attributes for SOA
     private List<Document> applicantSoaPack; // if personal service - CA/CB -> Either applicant pack //
-    //private List<Document> unservedRespondentPack; // if personal service - CA/CB -> Either applicant pack //
     private List<Document> respondentSoaPack;
     private String servedParty;
+    private String whoIsResponsible;
     private boolean wasCafcassServed;
+    private boolean wasCafcassCymruServed;
+    private boolean isPersonalService;
 
     private String orderType;
     private LocalDate createdDate;
-    private LocalDate servedDate;
+    private LocalDateTime servedDateTime;
     private boolean isNew;
```

Comment:
is this attribute needed here now since we have citizen notifications?

Reply:
Hi @yogendraU, It's needed here to see when the individual order was served.